stories/setup/storybook.override.styles.scss

```diff
@@ -0,0 +1,37 @@
+@import '~amo/css/styles';
+
+html,
+body {
+  // This overrides the App background style which makes it hard to see
+  // in storybook.
+  background: #fff;
+  // Giving extra cushion so nothing gets covered.
+  padding: 24px 12px 12px;
+}
+
+// This overrides some chapter styles.
+.section-container {
+  border-bottom: 1px solid $background-grey;
+  display: block;
+  // We need to use !important here to override chapter's inline style.
+  margin-bottom: 0 !important;
```

Comment:
The need for `!important` seems kind of fragile. I noticed that the chapters add-on has `useTheme`, so, would it be easier to make a custom theme? I can't find info on theming in the docs so maybe you already looked into this?

Reply:
yes good catch! actually I thought I had this set to off but 1 file didn't have this set correctly and so I missed this. Anyway, when I move it to config and set useTheme correctly, it does seem to remove the inline styles so I can remove these !importants :)